mlir/test/Dialect/Vector/linearize.mlir

```diff
@@ -1,17 +1,25 @@
 // RUN: mlir-opt %s -split-input-file -test-vector-linearize | FileCheck %s
+// RUN: mlir-opt %s -split-input-file -test-vector-linearize=target-vector-bitwidth=128 | FileCheck %s
```

Comment:
We should have a specific CHECK prefix for this

Reply:
Fixed